fontc/src/lib.rs

```diff
@@ -347,6 +348,39 @@ fn add_fvar_be_job(
     Ok(())
 }
 
+fn add_gvar_be_job(
+    change_detector: &mut ChangeDetector,
+    workload: &mut Workload,
+) -> Result<(), Error> {
+    let glyphs_changed = change_detector.glyphs_changed();
+
+    // If no glyph has changed there isn't a lot of merging to do
+    if !glyphs_changed.is_empty() {
+        let mut dependencies: HashSet<_> = glyphs_changed
+            .iter()
+            .map(|gn| BeWorkIdentifier::GvarFragment(gn.clone()).into())
+            .collect();
+        dependencies.insert(FeWorkIdentifier::FinalizeStaticMetadata.into());
```

Comment:
you'll probably need to add the glyph metrics to be able to compute phantom points

Reply:
Per-glyph work now emits glyph and gvar fragments, and pushes phantom points into the deltas. `leftSideX = glyph.xMin - leftSideBearing` seems like it would always be 0 since I believe we set lsb = xMin?